aiven/client/speller.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021, Aiven, https://aiven.io/
+#
+# This file is under the Apache License, Version 2.0.
+# See the file `LICENSE` for details.
+
+# Spelling Corrector in Python 3; see http://norvig.com/spell-correct.html
+#
+# Copyright (c) 2007-2016 Peter Norvig
+# MIT license: www.opensource.org/licenses/mit-license.php
+
+
+def suggest(word_to_check, word_dict):
```

Comment:
There should be a 3rd party lib with similar functionality implemented. The benefit of such lib is that it is up to date and maintained as opposed to blogpost code.

Reply:
Please propose a stable lightweight (without dictionaries included) library, as I didn't find any. Also, it's possible that we will start using a similar approach on the server-side, hence it would be great to have a Fedora RPM for that library.